lib/stimulus_reflex/reflex.rb

```diff
@@ -129,7 +129,13 @@ def params
     @_params ||= ActionController::Parameters.new(request.parameters)
   end
 
-  def dom_id(record_or_class, prefix = nil)
-    "#" + ActionView::RecordIdentifier.dom_id(record_or_class, prefix).to_s
+  def dom_id(record, prefix = nil)
+    return "#" + record.class.to_s.split("::")[0].pluralize.underscore if record.class < ActiveRecord::Relation
```

Comment:
Maybe you could simplify this to `record.table_name`

Reply:
I was actually wondering about that! The reason I didn't was because I wondered if there could be issues if the model class specified a non-standard table name. Is that a thing?

I spent a long time looking at `record.methods - Object.methods` trying to spot something that would return exactly what I needed, and saw `table_name` but was concerned:

```rb
class Task < ActiveRecord::Base
  self.table_name = 'projects'
end	end
```

Reply:
This is a thing, but `table_name` should reflect that if you overwrite the table name in the model.

Reply:
You're saying that even if I change my User model's underlying table to be backed by "people", that calling User.first.table_name will still return "user"?

Because what we need here is undoubtedly "user".

Reply:
Ah right, I understood it the other way around.

Okay in that case, this could work:
```ruby
record.model.model_name.collection
# or
record.model.model_name.plural
```

Reply:
I don't know what I'd do without you, @marcoroth!